package.json

```diff
@@ -14,8 +14,9 @@
     "lint": "eslint . --quiet --no-color && sass-lint --verbose",
     "release": "./scripts/release.sh",
     "release-beta": "./scripts/release-beta.sh",
-    "test": "BABEL_ENV=test mocha --opts scripts/mocha.opts --reporter dot && npm run lint",
-    "test:watch": "BABEL_ENV=test mocha --opts scripts/mocha.opts --reporter min --watch",
+    "test": "BABEL_ENV=test mocha --opts scripts/mocha.opts --compilers js:babel/register --reporter dot && npm run lint",
+    "test:coverage": "BABEL_ENV=test babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- --opts scripts/mocha.opts --reporter dot && npm run lint",
```

Comment:
Not with `babel-node` that is needed for it to work. (Trust me, I've tried reducing the commands as much as possible)
Here is the error when I try without it:
```txt
Error: Cannot find module 'path_to_my_project/babel-istanbul'

```